bitnami/influxdb/templates/pvc-backup.yaml

```diff
@@ -2,26 +2,17 @@
 Copyright VMware, Inc.
 SPDX-License-Identifier: APACHE-2.0
 */}}
-
-{{- if and .Values.backup.enabled .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
-kind: PersistentVolumeClaim
-apiVersion: v1
-metadata:
-  name: {{ include "common.names.fullname" . }}-backups
-  namespace: {{ include "common.names.namespace" . | quote }}
-  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
-    app.kubernetes.io/component: influxdb
-  {{- if or .Values.persistence.annotations .Values.commonAnnotations }}
-  {{- $annotations := include "common.tplvalues.merge" ( dict "values" ( list .Values.persistence.annotations .Values.commonAnnotations ) "context" . ) }}
-  annotations: {{- include "common.tplvalues.render" ( dict "value" $annotations "context" $) | nindent 4 }}
+{{-
+/*
+Prefer .Values.backup.persistence, but fall back to .Values.persistence if not present.
+*/
+-}}
+{{- if .Values.backup.enabled }}
+  {{- if and .Values.backup.persistence .Values.backup.persistence.enabled }}
```

Comment:
I see you are trying to keep backwards compatibility in case some user does not define the new `backup.persistence` block... But note that you are defining that block in the `values.yaml`. Those values are the default ones. So I'm not sure if we actually need to cover that case... Does it make sense?

Reply:
Whoops!

Would it make more sense to set the default value of `backup.persistence.enabled` to `false` so that users don't end up with an additional volume provisioned that they weren't expecting?

Reply:
The main reason I covered the case was that I assumed that people might be running these backups on the existing volume (though, I suppose that depends on a lot of factors, like whether or not their environment supports ReadWriteMany mode for the bound volume, and is binding that volume for both the database Pod and the backup Job)